canvas_modules/common-canvas/src/common-properties/ui-conditions/ui-groups-parser.js

```diff
@@ -44,6 +44,7 @@ function parseUiItem(panelTree, uiItem, currentPanel) {
 	case ItemType.PANEL:
 	case ItemType.CUSTOM_PANEL:
 	case ItemType.ADDITIONAL_LINK:
+	case ItemType.TEARSHEET:
```

Comment:
After this method runs, `this.panelTree` in properties-controller is set to this - 
![Screen Shot 2022-10-11 at 3 10 38 PM](https://user-images.githubusercontent.com/25124000/195213098-c4ca6bc3-76f1-4781-97bb-e0b0f052c9b0.png)